how-to-respond-en.md

```diff
@@ -0,0 +1,43 @@
+| <a href="README.md">Home</a> | <a href="introduction-en.md">Introduction</a> | <a href="introduction-en.md">Our proposals and questions</a> | How to respond |
```

Comment:
<a href="introduction-en.md">Our proposals and questions</a>

Link to be fixed